core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -176,9 +176,12 @@ enum class UniqueType(val text:String, vararg targets: UniqueTarget, val flags:
     EnablesResearchAgreements("Enables Research agreements", UniqueTarget.Global),
     TriggersVictory("Triggers victory", UniqueTarget.Global),
     TriggersCulturalVictory("Triggers a Cultural Victory upon completion", UniqueTarget.Global),
-
+    
+    CannotBuildUnits("Cannot build [baseUnitFilter] units", UniqueTarget.Nation),
```

Comment:
Why not Global? If you can build a building that grants you bonuses but prevents certain units that's an interesting trade off!